omero/sysadmins/unix/server-linux-walkthrough.txt

```diff
@@ -60,26 +62,69 @@ the omero user.
 
 .. literalinclude:: walkthrough/step04_all_omero.sh
 
-:download:`step04_all_omero.sh <walkthrough/step04_all_omero.sh>`
+
+Installing a web server
+-----------------------
+To Deploy OMERO.web, you can use either Nginx or Apache.
+Follow the steps to install your chosen web server.
+
 
 
-Installing Nginx
-----------------
+Nginx
+^^^^^
 
-As **root** install Nginx, copy the Nginx OMERO configuration file into the
+**The following steps are run as root.**
+
+Install Nginx, copy the Nginx OMERO configuration file into the
 Nginx configuration directory, and disable the default configuration:
 
 .. literalinclude:: walkthrough/step05_ubuntu1404_nginx.sh
 
 | :download:`step05_ubuntu1404_nginx.sh <walkthrough/step05_ubuntu1404_nginx.sh>`
-| :download:`step05_centos6_nginx.sh <walkthrough/step05_centos6_nginx.sh>`
+
+Apache
+^^^^^^
+
+**The following steps are run as root unless specified.**
+
+Install Apache 2.4::
+
+	apt-get -y install apache2 libapache2-mod-wsgi
+
+As the **omero** system user, configure OMERO.web by running the following commands:
+	
+.. literalinclude:: walkthrough/setup_omero_apache24.sh
 
 
+Copy the Apache OMERO configuration file into the
+Apache configuration directory, and disable the default configuration::
+
+    cp ~omero/OMERO.server/apache.conf.tmp /etc/apache2/sites-available/omero-web.conf
+    a2dissite 000-default.conf
+    a2ensite omero-web.conf
+
+    service apache2 start
+
+| :download:`step05_ubuntu1404_apache24.sh <walkthrough/step05_ubuntu1404_apache24.sh>`
+| :download:`setup_omero_apache24.sh <walkthrough/setup_omero_apache24.sh>`
+
+.. note::
+        The default value set for the ``WSGISocketPrefix`` directive in ``apache.conf.tmp``
+        needs to be modified:
+
+        ::
+
+            #see https://code.google.com/p/modwsgi/wiki/ConfigurationIssues
+            WSGISocketPrefix run/wsgi
+            # Use this on Ubuntu/Debian systems:
+            # WSGISocketPrefix /var/run/wsgi
```

Comment:
Since there's a dedicated script for Ubuntu you could script this change
https://github.com/ome/omero-install/blob/develop/linux/install_ubuntu1404_apache24.sh

E.g. `sed -i -r -e 's|(WSGISocketPrefix run/wsgi)|#\1|' -e 's|# (WSGISocketPrefix /var/run/wsgi)|\1|' apache.conf.tmp`